_topic_maps/_topic_map.yml

```diff
@@ -2997,6 +2997,8 @@ Topics:
       File: virt-schedule-vms
     - Name: Configuring PCI passthrough
       File: virt-configuring-pci-passthrough
+    - Name: Configuring vGPU passthrough for virtual machines
```

Comment:
I think it should be `Configuring vGPU passthrough` like `Configuring PCI passthrough` above, WDYT?

Reply:
Good call. I thought about that too, for consistency.